### ISSUE TITLE
Cannot read property 'slice' of undefined error

### ISSUE THREAD

**Issue by etherz10** [avh4/elm-upgrade#87]
```
$ npx elm-upgrade@latest
npx: installed 50 in 7.874s
(node:66389) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'slice' of undefined
    at /Users/kieron/.npm/_npx/66389/lib/node_modules/elm-upgrade/upgrade.js:154:64
    at Array.forEach (<anonymous>)
    at main (/Users/kieron/.npm/_npx/66389/lib/node_modules/elm-upgrade/upgrade.js:153:17)
    at /Users/kieron/.npm/_npx/66389/lib/node_modules/elm-upgrade/upgrade.js:575:7
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
(node:66389) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:66389) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Any ideas?

**Comment by nwolff** [comment]
dupe of #86 ?

**Comment by avh4** [comment]
Fixed in 0.19.8
